libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -94,8 +94,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
-    // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:UseBest,2:Blend,3:UseSecond
+    // @Description: Automatic switchover to GPS reporting best lock. SwitchOnHigherStatus swtiches to GPS reporting higher lock status, so less swtiching than UseBest. SwitchOnHigherStatus also helps to switch if primary lost GPS.
```

Comment:
If we want to add a description of what the values do, we should do all of them, not just one.

Reply:
I may add description for UseBest too. All other options are self-explanatory that is why I added some details for the new option.